CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+* `crud.len()` function to iterate over each shard and count space size
```

Comment:
`count space size`. The same. Len for vinyl has a bit different name - " maximum approximate number of tuples in the space".

https://www.tarantool.io/en/doc/latest/book/box/engines/